aggregator/src/app/AggregationStrategy.ts

```diff
@@ -235,11 +235,16 @@ export default class AggregationStrategy {
       let success: boolean;
 
       if (bundleResult.success) {
-        const [operationResults] = bundleResult.returnValue;
+        const [operationStatuses, results] = bundleResult.returnValue;
```

Comment:
When there is a failure this results variable has some data in it.  I haven't been able to decode that value though.  I get an out of bounds error everytime I try.  If anyone knows if I can decode that let me know.  Or if it is even worth trying to decode the value.

Reply:
I wouldn't worry about decoding the results for this work as we have https://github.com/web3well/bls-wallet/issues/364 to address this. Would focus on bubbling up human readable errors from submitted bundles.